packages/frontend/src/components/create-campaign/form/steps/pool-step/preview/index.tsx

```diff
@@ -5,24 +5,31 @@ import { PoolRemoteLogo } from "@/src/components/pool-remote-logo";
 
 import styles from "./styles.module.css";
 
-export function PoolStepPreview({ chainId, tokens, fee, usdTvl }: AmmPool) {
+export function PoolStepPreview({
+    chainId,
+    tokens,
+    fee,
+    usdTvl,
+}: Partial<AmmPool>) {
```

Comment:
Why the partial here?

Reply:
Addressed in 245569fa20727fc0c5176810a8cb2388b8730b78.